package.json

```diff
@@ -131,6 +131,7 @@
     "luxon": "1.21.3",
     "markdown-it": "10.0.0",
     "markdown-table": "1.1.3",
+    "micromatch": "4.0.2",
```

Comment:
What caused your preference for micromatch over minimatch?

Reply:
When searching for glob matchers, the description of micromatch in the NPM registry pitched itself as the more performant alternative to minimatch. I didn't realise that minimatch was already a dependency in Renovate, otherwise I would have just stuck with that. I've reverted back to using minimatch and removed micromatch as a dependency